subprojects/packagefiles/sdl2/meson_options.txt

```diff
@@ -1,3 +1,16 @@
+option(
+    'test',
+    type : 'boolean',
+    value : 'false',
+    description : 'build tests'
+)
+
+option(
+    'run_test',
+    type : 'boolean',
+    value : 'false',
```

Comment:
AFAIK it is uncommon to have separate option to run tests, is it really needed? I would at least make it `true` by default.

Reply:
It is, but I would rather not run these by default
the tests create windows, do audio playback, etc. and do not automatically terminate.

Reply:
By default you already don't even build them, that seems good enough, no?

Reply:
I guess?

I find compiling the tests more useful than actually running them because it shows if anything is missing, but I can merge the two and keep tests off by default

Reply:
That's true. Maybe set `tests` to true by default, and `run_tests` false? Just small nitpicking anyway ;P

Reply:
There's a pretty simple solution, you can use `add_test_setup(..., is_default: true)` in combination with setting suites for these tests, to only run a subset of tests with the default `meson test`.